$NetBSD$

Use FLTK 1.4 for glyph substitution on X11 platform.
Modify detection for OpenSSL 1.1.

--- configure.ac.orig	2015-06-30 14:07:06.000000000 +0000
+++ configure.ac
@@ -107,15 +107,15 @@ fi
 
 
 dnl -------------------------
-dnl Test for FLTK 1.3 library
+dnl Test for FLTK 1.4 library
 dnl -------------------------
 dnl
 dnl For debugging and to be user friendly
-AC_MSG_CHECKING([FLTK 1.3])
+AC_MSG_CHECKING([FLTK 1.4])
 AC_PATH_PROG(FLTK_CONFIG,fltk-config)
 fltk_version="`$FLTK_CONFIG --version 2>/dev/null`"
 case $fltk_version in
-  1.3.*) AC_MSG_RESULT(yes)
+  1.4.*) AC_MSG_RESULT(yes)
          LIBFLTK_CXXFLAGS=`$FLTK_CONFIG --cxxflags`
          LIBFLTK_CFLAGS=`$FLTK_CONFIG --cflags`
          LIBFLTK_LIBS=`$FLTK_CONFIG --ldflags`;;
@@ -286,7 +286,7 @@ if test "x$enable_ssl" = "xyes"; then
 
   if test "x$ssl_ok" = "xyes"; then
     old_libs="$LIBS"
-    AC_CHECK_LIB(ssl, SSL_library_init, ssl_ok=yes, ssl_ok=no, -lcrypto)
+    AC_CHECK_LIB(ssl, OPENSSL_config, ssl_ok=yes, ssl_ok=no, -lcrypto)
     LIBS="$old_libs"
   fi
 
